questions/qa-international-multilingual.zh-hans.html

```diff
@@ -0,0 +1,106 @@
+f.lastSubstUpdate = { date:'2004-04-30', time:'14:25'}  // date and time of latest substantive changes to this document
+f.status = 'published'  // should be one of draft, review, published, notreviewed or obsolete
+f.path = '../' // what you need to prepend to a URL to get to the /International directory
+
+// AUTHORS AND TRANSLATORS should fill in these assignments:
+f.thisVersion = { date:'2022-04-13', time:'07:36'} // date and time of latest edits to this document/translation
+f.contributors = '' // people providing useful contributions or feedback during review or at other times
+// also make sure that the lang attribute on the html tag is correct!
+f.sources = '' // describes sources of information
+
+// TRANSLATORS should fill in these assignments:
+f.translators = '<a href="http://cuihuan.net" title="Huan Cui">Huan Cui</a>' // translator(s) and their affiliation - a elements allowed, but use double quotes for attributes
+
+f.breadcrumb = 'other'
+
+f.additionalLinks = ''
+</script>
+<script src="qa-international-multilingual-data/translations.js"> </script>
+<script src="../javascript/doc-structure/article-dt.js"> </script>
+<script src="../javascript/boilerplate-text/boilerplate-zh-hans.js"></script><!--TRANSLATORS must change -en to the subtag for their language!-->
+<script src="../javascript/doc-structure/article.js"> </script>
+<script src="../javascript/articletoc-html5.js"></script>
+<link rel="stylesheet" href="../style/article-2016.css" />
+<link rel="copyright" href="#copyright"/>
+<!--[if lt IE 9]><script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
+</head>
+
+<body>
+<header>
+  <nav id="mainNavigation"></nav><script>document.getElementById('mainNavigation').innerHTML = mainNavigation</script>
+
+  <h1>国际化和多语言网站</h1>
+</header>
+
+
+<section id="sidebarExtras">
+  <section>
+    <h2 id="links" class="notoc"><a href="#links">相关链接</a></h2>
+    <p><a class="print" href="/International/questions/qa-mono-multilingual">单语言和多语言网站</a></p>
+  </section>
+</section>
+
+
+<section>
+  <div id="audience">
+    <p><span id="intendedAudience" class="leadin">目标受众:</span>
+    任何想了解“国际化”网站或“多语言”网站区别的人</p>
+    <div id="updateInfo"></div><script>document.getElementById('updateInfo').innerHTML = g.updated</script>
+    </div>
+
+  <h2 id="question"><a href="#question">问题</a></h2>
+    <p class="question">如何定义“国际化”或“多语言”网站？</p>
+    <p>“国际化”或“多语言”两个概念在部分场景下是可互换的，但是，当用来定义网站的时候，他们有非常明确的含义。清楚地区分这两个概念很重要。理解二者的差异，可以帮助您更准确的规划和定义网站需求。同时，二者在设计、开发和部署阶段都有非常明确的要求，这些要求通常需要不同的技术解决方案。
+    </p>
+</section>
+
+
+<section>
+  <h2 id="answer"><a href="#answer">答案</a></h2>
+    <p>简而言之，“国际化”是指网站面向全球用户，而“多语言”是指网站不止一种语言。国际化网站可以使用多种语言，也可以是单语言；同样的，多语言的网站可以面向全球用户，也可以仅面向本地用户。举个例子，W3C的站点是个国际化站点，因为他是为全球受众设计的。尽管该网站主要用英语编写，但有些页面包含其他语言，因此该站点也是个多语言网站。</p>
+    <p>多数公司的全球主页是原生支持“国际化”的。由于要面向全球用户，网站通常要求用户选择一个区域站点。此类区域站点往往使用单一语言的（通常是当地主要目标用户的通用语言）。选择区域后，会将用户重定向到同域名下的一个子站点或者当前区域的独立网站。无论哪种方式，良好的一致性设计，都会让网站的切换不影响用户体验，同时，保留国际站点的能力。</p>
```

Comment:
```suggestion
    <p>多数公司的全球主页是原生支持“国际化”的。由于要面向全球用户，网站通常要求用户选择一个区域站点。此类区域站点往往使用单一语言的（通常是当地主要目标用户的通用语言）。选择区域后，会将用户重定向到同域名下的一个子站点或者当前区域的独立网站。无论采用何种方式，良好的一致性设计都会让用户觉得它仍然是同一个站点的一部分，并保留国际站点的感觉。</p>
```

「不影响用户体验」和「保留国际站点的**能力**」和原文意思不符，感觉有点过于意译了。